src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -290,25 +290,26 @@ protected function sortServiceIds(array $serviceIds)
     /**
      * Gets class description from a docblock.
      *
-     * @param string $class
-     *
-     * @return string
+     * @internal
```

Comment:
the whole sub-namespace is internal, can be removed

Reply:
thanks, removed